### ISSUE TITLE
GUI : FED : Hardware status expand button accessiblity

### ISSUE THREAD

**Issue by yoshiemuranaka** [ibm-openbmc/dev#2405]
## Overview
The expand buttons in tables on the hardware status page have accessibility violations.

> Form control element <button> has no associated label
> Each form control must have an associated label
> IBM 3.3.2 Labels or Instructions | WCAG 2.1 technique H44 | WCAG 2.1 technique ARIA16

![Screen Shot 2020-07-22 at 9.31.05 AM.png](https://images.zenhubusercontent.com/5cbe076507507b08996e895c/22f93188-d2af-492d-9ef4-2656555ef5ad)

**Comment by derick-montague** [comment]
My suggestion for resolving this is to add instructional text that is screen reader only text within the button.

**Comment by dixsie** [comment]
resolved by https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/36306
